src/client.ts

```diff
@@ -110,19 +110,26 @@ export class AutoblocksAPIClient {
   }
 
   private async get<T>(path: string): Promise<T> {
-    const resp = await fetch(`${API_ENDPOINT}${path}`, {
+    const url = `${API_ENDPOINT}${path}`;
+    const resp = await fetch(url, {
       method: 'GET',
       headers: {
         ...AUTOBLOCKS_HEADERS,
         Authorization: `Bearer ${this.apiKey}`,
       },
       signal: AbortSignal.timeout(this.timeoutMs),
     });
+    if (!resp.ok) {
```

Comment:
this should have been added when we switched away from axios (axios would throw for you but for regular fetch we need to raise ourselves). similar to `raise_for_status` in python